src/main.rs

```diff
@@ -54,10 +60,15 @@ fn exec_new(args: &ArgMatches) {
         .expect("Bug: project_name is required");
     let project_name = project_name.to_owned();
     let version = args.value_of("amethyst_version").map(ToOwned::to_owned);
+    let no_defaults = match args.occurrences_of("no_defaults") {
```

Comment:
This should be `args.is_present("no_defaults")`

Reply:
Oh, you are right, this is much better! Fixing :)